common/remote/sockfile.cpp

```diff
@@ -3016,25 +3016,27 @@ static CriticalSection ClientCountSect;
 
 class CClientStats : public CInterface
 {
-    SpinLock spin;
 public:
-    CClientStats(const char *_client) : client(_client) { count = 0; bRead = 0; bWritten = 0; }
+    CClientStats(const char *_client) : client(_client), count(0), bRead(0), bWritten(0) { }
     const char *queryFindString() const { return client; }
-    inline void addRead(unsigned len)
```

Comment:
The initial trigger for this change was the parameter wasn't wide enough - causing a restriction cast on one of the calls.  However it made sense to reimplement using atomics as per the comments